ui/src/components/Rules/RuleGroup/index.tsx

```diff
@@ -47,50 +47,53 @@ const RuleGroup = (props: {
   }
 
   return (
-    <div className="relative mb-5 flex w-full flex-col overflow-hidden rounded-xl bg-zinc-800 bg-cover bg-center p-4 text-zinc-400 shadow ring-1 ring-zinc-700 sm:flex-row">
-      <div className="relative z-10 flex w-full min-w-0 flex-col pr-4 sm:w-5/6 sm:flex-row">
-        <div className="mb-3 flex flex-col sm:mb-0 sm:w-5/6">
-          <div className="flex text-xs font-medium text-white">
-            <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-base font-bold text-white sm:text-lg">
-              {props.group.name}
-            </div>
+    <div className="flex-auto">
+      <div className="text-base font-bold text-white sm:text-lg">
+        {props.group.name}
+      </div>
+      <div className="mt-2 h-28 overflow-hidden overflow-y-auto overflow-ellipsis font-semibold">
+        {props.group.description}
+      </div>
+      <div className="mb-2 mt-2 grid grid-cols-3 gap-3">
+        <div>
+          <div className="align-center flex justify-center text-sm font-medium">
+            Status
           </div>
-
-          <div className="my-0.5 flex text-sm sm:my-1">
-            <span className="mr-2 w-full max-w-prose overflow-hidden overflow-ellipsis font-semibold">
-              {props.group.description}
-            </span>
+          <div>
+            {props.group.isActive ? (
+              <span className="flex justify-center font-semibold text-green-600">
```

Comment:
To match the collections page:
```suggestion
              <span className="flex justify-center text-green-600">
```

Reply:
The font sizes are also different across the two pages

Reply:
I am not sure I changed that. I mean Github says I did, but I don't remember looking at those things. I will update the PR to make it match the collections page, as I don't like the semibold on the description. I prefer the collection page's style, now that I am looking at that.

Reply:
Should be fixed.